x/programs/cmd/simulator/cmd/plan_test.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateCallParams(t *testing.T) {
+	// Using a key not yet created must return an error
+	// This test creates a simulator to initialize the db we need to use
+	ctx := context.Background()
+	newSimulator := func() *Simulator {
+		logLevel := "info"
```

Comment:
Can we please avoid printing to the terminal when running the test ?

```suggestion
		logLevel := "error"
```

Reply:
It should print to stdout, not stderr by the way but we will fix this later.

Reply:
Why do we want to potentially print logs into stdout? Stdout already prints `Response` object into stdout. Mixing both we will not be able to Parse stdout to analyse result when running the simulator.

Reply:
Communicating over stdin/out is an anti-pattern and we want to avoid that, we worked towards building an abstract layer of communication and will switch to grpc and / or ipc later on

Reply:
Ok it makes sense 👍